src/ResourceManager/Sql/Commands.Sql/help/New-AzureRmSqlDatabaseCopy.md

```diff
@@ -143,7 +143,7 @@ Accept wildcard characters: False
 ```
 
 ### -ResourceGroupName
-Specifies the name of the  Resource Group to which this cmdlet assigns the copied database.
+Specifies the name of the  Resource Group that contains the database to copy.
```

Comment:
Nit: there is an extra space between "the" and "Resource Group"

Reply:
Is this update correct? Looking at the code I get this impression:
copyResourceGroup = Source DB RG
ResourceGroup = Target DB RG
https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Sql/Commands.Sql/Replication/Cmdlet/NewAzureSqlDatabaseCopy.cs

Reply:
I believe that update is correct - copyResourceGroup is a resource group of a target and ResourceGroup is a resource group of source